.github/workflows/ci-workflow.yaml

```diff
@@ -8,15 +8,13 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [16.x, 18.x]
```

Comment:
Let's run two tests in parallel to keep up with https://github.com/nodejs/Release#nodejs-release-working-group